server/web/auth/sendemail.py

```diff
@@ -0,0 +1,21 @@
+from flask_mail import Message, Mail
```

Comment:
Move file to under `web/` so we have easier access to the functions and rename it to `utils.py`